examples/BEMIO/CAPYTAINE/run_cases.py

```diff
@@ -0,0 +1,50 @@
+# '-*- coding: utf-8 -*-
+"""
+Created on Wed Nov 18 11:32:31 2020
+
+@author: akeeste
+
+This script runs all of the current cases for WEC-Sim bemio examples.
+
```

Comment:
This script runs Capytaine for each case not BEMIO. I included this and call_capytaine.py so that everyone can see how Capytaine is being run for each case because there is an additional function required (hydrostatics)